scripts/generate_thumbnail.py

```diff
@@ -32,12 +32,16 @@
 with open(data_path, 'rt', encoding='utf-8') as f:
     data = json.loads(f.read())
 
+print('nopeeee')
```

Comment:
What's with all of the `print` commands?

Reply:
Ah damnit I thought I remove it all ! I'll remove them

Reply:
I made a quick fix